src/extensions/ContentDiff/shared/helpers.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Separate these tests out as well to match the split in the helpers comments above